server/webhookHandlers/handleItemWebhook.js

```diff
@@ -24,23 +24,22 @@ const itemErrorHandler = async (plaidItemId, error) => {
       await updateItemStatus(itemId, 'bad');
       break;
     }
-
     // TODO: handle item error webhook cases
-    case 'INVALID_CREDENTIALS':
-    case 'INVALID_MFA':
-    case 'INVALID_UPDATED_USERNAME':
-    case 'ITEM_LOCKED':
-    case 'ITEM_NO_ERROR':
-    case 'ITEM_NOT_SUPPORTED':
-    case 'ITEM_NO_VERIFICATION':
-    case 'INCORRECT_DEPOSIT_AMOUNTS':
-    case 'TOO_MANY_VERIFICATION_ATTEMPTS':
-    case 'USER_SETUP_REQUIRED':
-    case 'MFA_NOT_SUPPORTED':
-    case 'NO_ACCOUNTS':
-    case 'NO_AUTH_ACCOUNTS':
-    case 'PRODUCT_NOT_READY':
-    case 'PRODUCTS_NOT_SUPPORTED':
+    // case 'INVALID_CREDENTIALS':
```

Comment:
are these cases not needed anymore? can we just get rid of them?

Reply:
yes...got rid of them